src/components/expansionPanel.js

```diff
@@ -0,0 +1,33 @@
+(() => ({
+  name: 'ExpansionPanel',
+  category: 'LAYOUT',
+  type: 'CONTAINER_COMPONENT',
+  allowedTypes: ['BODY_COMPONENT', 'CONTAINER_COMPONENT', 'CONTENT_COMPONENT'],
+  orientation: 'HORIZONTAL',
+  jsx: (() => {
+    const {
+      ExpansionPanel,
+      ExpansionPanelSummary,
+      ExpansionPanelDetails,
+      Typography,
+    } = window.MaterialUI.Core;
+    const { ExpandMore } = window.MaterialUI.Icons;
+    const isDev = B.env === 'dev';
+    const { title, disabled } = options;
+    const ExpansionPanelComponent = (
+      <ExpansionPanel disabled={disabled}>
+        <ExpansionPanelSummary expandIcon={<ExpandMore />}>
+          <Typography>{title}</Typography>
```

Comment:
For now it's easier to only accept text, and not children. Because it makes the component more complex. Split it into panel, summary, details. Now it's just one

Reply:
Should we than uphold the same logic for the card component to make it more consistent?

Reply:
This is only for the summary. The details already has the same setup as Card